src/pcapi/core/bookings/validation.py

```diff
@@ -96,6 +96,17 @@ def check_offerer_can_cancel_booking(booking: Booking) -> None:
         raise forbidden
 
 
+def check_system_can_cancel_expired_booking(booking: Booking) -> None:
+    if not booking.stock.offer.offerType["canExpire"]:
+        raise exceptions.DoesNotExpire(
+            "Booking %s is of type %s, which does not expire", booking, booking.stock.offer.type
+        )
+    if booking.isCancelled:
+        raise exceptions.AlreadyCancelled("Booking %s is already canceled", booking)
```

Comment:
```suggestion
        raise exceptions.AlreadyCancelled("Booking %s is already cancelled", booking)
```
les deux se disent donc peu importent lequel mais il faudrait qu'on ai le même partout ^^ (pas bloquant)

Reply:
Je choisis l'orthographe non-US alors: `cancelled` !